inject.js

```diff
@@ -385,22 +394,22 @@
   }
 
   function runAction(action, document, value, e) {
-    var videoTags = document.getElementsByTagName('video');
+    if (tc.settings.audioBoolean) {
+      var videoTags = document.querySelectorAll('video,audio');
+    } else {
+      var videoTags = document.querySelectorAll('video');
+    }
+	console.log(videoTags)
     videoTags.forEach = Array.prototype.forEach;
 
     // Get the controller that was used if called from a button press event e
-    if (e){
+    if (e) {
       var targetController = e.target.getRootNode().host;
-    }
+    } 
 
     videoTags.forEach(function(v) {
       var id = v.dataset['vscid'];
       var controller = document.querySelector(`div[data-vscid="${id}"]`);
-
-      // Don't change video speed if the video has a different controller
-      if (e && !(targetController == controller)) {
```

Comment:
This was just added. Are you intentionally removing it?

Reply:
No. I was trying to undo the addition of the code in my pull request #452